src/test/func/malloc/malloc.cc

```diff
@@ -138,6 +138,23 @@ void test_memalign(size_t size, size_t align, int err, bool null)
   check_result(size, align, p, err, null);
 }
 
+#if defined(__NetBSD__)
+void test_reallocarr(void* p, size_t nmemb, size_t size, int err, bool null)
+{
+  printf("reallocarr(%zu, %zu)\n", nmemb, size);
+  errno = SUCCESS;
+  int r = our_reallocarr(p, nmemb, size);
```

Comment:
This doesn't look like the correct use case.  The Man page gives the following use case:
```C
           int *data = NULL;
           int ret = 0;

           ret = reallocarr(&data, INITSIZE, sizeof(*data));
           if (ret)
               errc(1, ret, "reallocarr failed");

           ret = reallocarr(&data, NEWSIZE, sizeof(*data));
           if (ret)
               errc(1, ret, "reallocarr failed on resize");
```
https://man.netbsd.org/reallocarr.3

Reply:
good catch